osu.Game/Overlays/News/NewsArticleCover.cs

```diff
@@ -0,0 +1,159 @@
+                    Colour = ColourInfo.GradientVertical(Color4.Black.Opacity(0.1f), Color4.Black.Opacity(0.6f)),
+                    Alpha = 1f,
+                },
+                new DateContainer(info.Time)
+                {
+                    Margin = new MarginPadding
+                    {
+                        Right = 20,
+                        Top = 20,
+                    }
+                },
+                new OsuSpriteText
+                {
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    Margin = new MarginPadding
+                    {
+                        Left = 25,
+                        Bottom = 50,
+                    },
+                    Font = OsuFont.GetFont(Typeface.Exo, 24, FontWeight.Bold),
+                    Text = info.Title,
+                },
+                new OsuSpriteText
+                {
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    Margin = new MarginPadding
+                    {
+                        Left = 25,
+                        Bottom = 30,
+                    },
+                    Font = OsuFont.GetFont(Typeface.Exo, 16, FontWeight.Bold),
+                    Text = "by " + info.Author
+                }
+            };
+
+            bg.OnLoadComplete += d => d.FadeIn(250, Easing.In);
+        }
+
+        //news article cover background
+        [LongRunningLoad]
+        private class NewsBackground : Sprite
+        {
+            private readonly string url;
+
+            public NewsBackground(string coverUrl)
+            {
+                url = coverUrl ?? "Headers/news";
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(LargeTextureStore store)
+            {
+                Texture = store.Get(url);
+            }
+        }
+
+        //date container
+        private class DateContainer : Container, IHasTooltip
+        {
+            private readonly DateTime date;
+
+            public DateContainer(DateTime date)
+            {
+                this.date = date;
+
+                Anchor = Anchor.TopRight;
+                Origin = Anchor.TopRight;
+                Masking = true;
+                CornerRadius = 4;
+                AutoSizeAxes = Axes.Both;
+                Children = new Drawable[]
+                {
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = Color4.Black.Opacity(0.5f),
+                    },
+                    new OsuSpriteText
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Font = OsuFont.GetFont(Typeface.Exo, 12, FontWeight.Black, false, false),
+                        Text = date.ToString("dd MMM yyy"),
```

Comment:
Don't know if we're after parity with web here, but if we are then this should probably be
```suggestion
                        Text = date.ToString("d MMM yyy").ToUpper(),
```